components/features/Signup/AssesseeSignup/AssesseeDetails/AssesseeDetails.tsx

```diff
@@ -52,17 +52,22 @@ const AssesseeDetails = () => {
     setValue("date_of_birth", dateStr);
   };
 
-  const continueNext = () => {
+  const continueNext: FormEventHandler<Element> = (e) => {
+    e.preventDefault()
+
     const [isFirstNameValid, firstNameError] = emptyValidator(data.first_name);
     setError("first_name", firstNameError);
 
     const [isLastNameValid, lastNameError] = emptyValidator(data.last_name);
     setError("last_name", lastNameError);
 
+    const [isPhoneNumberValid, phoneNumberError] = emptyValidator(data.phone_number);
```

Comment:
Hello, Alex. Will you also check phone number format correctness in the assessee's signup front end?

Reply:
Hello, Jo. Yes I am planning to do that, however, I may need to know how the correctness of a phone number is handled in the backend.

Reply:
Yes, Alex. It has been handled in the backend.